lib/career_bot/services/action_response_service.rb

```diff
@@ -0,0 +1,22 @@
+class ActionResponseService
```

Comment:
I would think about some other name for this class. This one is kinda messy, I think